src/components/Vuetable.vue

```diff
@@ -377,14 +377,13 @@ export default {
       this.httpOptions['params'] = this.getAllQueryParams()
 
       if (this.httpMethod.toLowerCase() == 'post'){
```

Comment:
cleaner way would be to 'validate' the request type, and then call it like:
```
axios[httpMethod](this.apiUrl, this.httpOptions.params).then(
```

save you writing the request up twice. other than that i like this. I prefer to use post request.